std_msgs/String sensor
sensor_msgs/PointCloud2[] patterns # different board locations for this sensor
